docs/iris/src/developers_guide/contributing_documentation.rst

```diff
@@ -46,11 +46,9 @@ If you wish to run a clean build you can run::
 
 This is useful for a final test before committing your changes.
 
-.. note:: In addition to the automated `travis-ci`_  build of the documentation,
-          the https://readthedocs.org/ service is also used.  The configuration
-          of this held in a file in the root of the
-          `github Iris project <https://github.com/SciTools/iris>`_  named
-          ``.readthedocs.yml``.
+.. note:: In order to preserve a clean build for the html, all **warnings**
+          have been promoted to be **errors** to ensure they are addressed.
+          This **only** applies when ``make html`` is run.
```

Comment:
BTW is it now best to in general do
```
.. note::

   <content>
```
rather than
```
.. note:: <content>
```
Just curious...

Reply:
Either is ok in my books.  I prefer the latter as it helps the note be more visual - but this is my preference.  If you have a particularly long note then the first option may be better as there is more text per line (due to no indentation)